ci/meta.yml

```diff
@@ -78,3 +78,89 @@ meta:
           - name: dw-al2-ecs-ami
       params:
         DETAILED_EXITCODE: -detailed-exitcode
+    test-ecs-ami:
+      task: test-ecs-ami
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: ((dataworks.docker_awscli_repository))
+            version: ((dataworks.docker_awscli_version))
+            tag: ((dataworks.docker_awscli_version))
+        run:
+          path: sh
+          args:
+            - -exc
+            - |
+              export AWS_DEFAULT_REGION
+              source /assume-role
+              set -x
+              max_retries=15
+              AMI="$(cat dw-al2-ecs-ami/id)"
+              CEs="$(aws batch describe-compute-environments)"
+
+              if echo $CEs | grep -q $AMI; then echo "Compute environment is using the expected ami";
+              else echo "No compute environment is associated with the AMI to be tested"; exit 1;
+              fi
+
+              CEs=$(aws batch describe-compute-environments --query 'computeEnvironments[]' \
+              | jq -r '.[] | select(.computeResources.imageId=='\"$AMI\"') | .computeEnvironmentName')
+
+              CE="$(echo $CEs | grep -o -e $COMPUTE_ENV_PREFIX'[a-zA-Z0-9_-]*')"
+
+              job_id=$(aws batch submit-job --job-name s3_object_tagger_test_ami_job \
+              --job-definition s3_object_tagger_test_ami_job --job-queue amitest_s3tagger | jq -r '.jobId')
+
+              while [ $max_retries > 0 ]
```

Comment:
I don't like `max_retries` being amended - it should really be a constant (the maximum number of retries doesn't change) and a count variable used to increment.